tests/test_publish.py

```diff
@@ -590,23 +585,23 @@ def test_publish_attachment(tmpdir):
             '1.0.0',
             '2.0.0',
             [],
-            ['extra/folder/file2.txt'],
+            [os.path.join('extra', 'folder', 'file2.txt')],
```

Comment:
Why do you have yo change this?
The path of the attachment should always (also on Windows) be encoded by `/` as we also do with media files.

Reply:
Because an attachment might be a folder, we compare against the result of `audeer.list_file_names()`.